src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -290,12 +305,21 @@ impl<S: S3BackEnd + Sync + Send + 'static> MetaData for S3MetaData<S> {
                     parent, parent_name
                 )
             });
-            (pnode.full_path().to_owned(), new_node_attr, fuse_attr)
+            (pnode.full_path().to_owned(), full_path, new_node_attr, fuse_attr)
         };
 
         self.sync_attr_remote(&parent_full_path).await;
         self.sync_dir_remote(&parent_full_path, node_name, &child_attr, target_path)
             .await;
+
+        // We dont need to sync for the unmark
+        // todo: use etcd lease for auto expire.
+        let etcd_client=Arc::clone(&self.etcd_client);
+        tokio::spawn(async move{
```

Comment:
What if something goes wrong in the task? Is there a way to check the state of the task?

Reply:
I haven't thought about this part. Maybe we should just use a lease.